nixos/modules/services/networking/wg-quick.nix

```diff
@@ -247,7 +259,7 @@ let
           optionalString (peer.allowedIPs != []) "AllowedIPs = ${concatStringsSep "," peer.allowedIPs}\n"
         ) values.peers;
       };
-      configPath = "${configDir}/${name}.conf";
+      configPath = if values.configFile != null then "/tmp/${name}.conf" else "${configDir}/${name}.conf";
```

Comment:
We should rather just something like StateDirectory instead of hardcoding /temp.

Reply:
mmmm, I am using /tmp for the reason it will get cleared after launch — I am not sure StateDirectory provides me with that, and that would mean additionally clearing every launch

and since /tmp is bound in the cgroup by PrivateTmp, it'll be on `/tmp/systemd-smth-smth-ifeawOJfew-wg-quick-quack` folder — so it's not as hardcoded as it seems

Reply:
I think we should add a comment that it isn't the global /tmp. Then this should be fine.

Reply:
ok